testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -4465,44 +4465,35 @@ void testBitGetFunc(SqlOperatorFixture f, String functionName) {
     f.checkString("to_char(timestamp '2022-06-03 12:15:48.678', 'YYYY-MM-DD HH24:MI:SS.MS TZ')",
         "2022-06-03 12:15:48.678",
         "VARCHAR NOT NULL");
-    f.checkString("to_char(timestamp '2022-06-03 12:15:48.678', 'Day')",
```

Comment:
why delete these tests?

Reply:
A mistake on my part. They have been added back, and I have added some more tests.